lte/gateway/python/magma/pipelined/tests/test_uplink_bridge.py

```diff
@@ -801,22 +825,145 @@ def test_flow_snapshot_match(self):
         subprocess.check_call(cmd1)
         assert_bridge_snapshot_match(self, self.BRIDGE, self.service_manager)
 
+#----
+# DHCP based test
+@unittest.skip
```

Comment:
why are we skipping this one?

Reply:
this depends on a thread running in uplink-br app. so the test is stuck in hung state. I need to figure out how to run the test in parallel.

Reply:
I will do it as followup.